docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -12,26 +12,36 @@ using the ``access-control.config-files`` configuration property. It should
 contain a comma separated list of the access control property files to use
 (rather than the default ``etc/access-control.properties``).
 
-Trino offers the following built-in implementations:
-
-================================================== =================================================================
-System access control name                         Description
-================================================== =================================================================
-``default``                                        All operations are permitted, except for user impersonation.
-                                                   This is the default access control if none are configured.
-
-``allow-all``                                      All operations are permitted.
-
-``read-only``                                      Operations that read data or metadata are permitted, but
-                                                   none of the operations that write data or metadata are allowed.
-
-``file``                                           Authorization rules are specified in a config file.
-                                                   See :doc:`file-system-access-control`.
-================================================== =================================================================
+Trino offers the following built-in system access control implementations:
+
+.. list-table::
+  :widths: 20, 80
+  :header-rows: 1
+
+  * - Name
+    - Description
+  * - ``default``
+    - All operations are permitted, except for user impersonation and triggering
+      :doc:`/admin/graceful-shutdown`.
+
+      This is the default access control if none are configured.
+  * - ``allow-all``
+    - All operations are permitted.
+  * - ``read-only``
+    - Operations that read data or metadata are permitted, but none of the
+      operations that write data or metadata are allowed.
+  * - ``file``
+    - Authorization rules are specified in a config file. See
+      :doc:`file-system-access-control`.
 
 If you want to limit access on a system level in any other way than the ones
 listed above, you must implement a custom :doc:`/develop/system-access-control`.
 
+Access control must be configured on the coordinator. However authorization for
+certain operations, such a triggering :doc:`/admin/graceful-shutdown`, are
+performed on specific worker nodes. In such cases access control must also be
```

Comment:
```suggestion
performed on specific worker nodes. In these cases, access control must also be
```
What other operations? I feel like "certain operations" is a little vague and unhelpful.

Reply:
Reworded the whole paragraph some more.